cmd/evmosd/testnet.go

```diff
@@ -324,7 +342,15 @@ func initTestnetFiles(
 			return err
 		}
 
+		const CreateValidatorTxGasLimit = 300000 // This transaction consumes approximately 220,000 gas when executed in the genesis block.
```

Comment:
Please, add this as a constant outside this function. Could be above `addTestnetFlagsToCmd` (L96).

Also, don't export this constant, rename it to:
```suggestion
                // createValidatorMsgGasLimit is the gas limit used in the MsgCreateValidator included in genesis transactions.
                // This transaction consumes approximately 220,000 gas when executed in the genesis block.
		const createValidatorMsgGasLimit = 250_000
```

Reply:
done